tests/fortran/view_test.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2023 ETH Zurich and the DaCe authors. All rights reserved.
+
+from fparser.common.readfortran import FortranStringReader
+from fparser.common.readfortran import FortranFileReader
+from fparser.two.parser import ParserFactory
+import sys, os
+import numpy as np
+import pytest
+
+from dace import SDFG, SDFGState, nodes, dtypes, data, subsets, symbolic
+from dace.frontend.fortran import fortran_parser
+from fparser.two.symbol_table import SymbolTable
+from dace.sdfg import utils as sdutil
+
+import dace.frontend.fortran.ast_components as ast_components
+import dace.frontend.fortran.ast_transforms as ast_transforms
+import dace.frontend.fortran.ast_utils as ast_utils
+import dace.frontend.fortran.ast_internal_classes as ast_internal_classes
+
+
+def test_fortran_frontend_view_test():
```

Comment:
If you have the time, it would be nice to have a brief, 1-sentence comment that says what the test is trying to check. Some test names are a bit cryptic sometimes :-)

Reply:
Done!

Reply:
Thank you very much!